src/assets/css/style.scss

```diff
@@ -0,0 +1,100 @@
+@import url('https://fonts.googleapis.com/css2?family=Raleway:ital@0;1&display=swap');
```

Comment:
- [ ] I'm sure you could improve your style to be more according to the [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) example: 🐱 
- Your work
![image](https://user-images.githubusercontent.com/20563252/125544018-8e16fff0-7fa6-490d-8428-2202f26ab7a3.png)
- The example
![image](https://user-images.githubusercontent.com/20563252/125544054-2839e048-9bdd-4272-b144-ef3fb432c0c2.png)